dev/ctests/cases/C48_ATM_gfs_fcst_seg0_cmp.yaml

```diff
@@ -0,0 +1,84 @@
+{% set H_offset = '-6H' %}
```

Comment:
what is the intent of this `_cmp` yaml?

Reply:
Those where to be used for cases that could be currently compared.  You recommended to remove them and advance the case file with subsection types that would drive the comparison regime.